src/api/server/sync_io.rs

```diff
@@ -19,10 +21,46 @@ use crate::abi::virtio_fs::{RemovemappingIn, RemovemappingOne, SetupmappingIn};
 use crate::api::filesystem::{
     DirEntry, Entry, FileSystem, GetxattrReply, IoctlData, ListxattrReply,
 };
+#[cfg(feature = "fusedev")]
+use crate::transport::FuseDevWriter;
 use crate::transport::{pagesize, FsCacheReqHandler, Reader, Writer};
 use crate::{bytes_to_cstr, encode_io_error_kind, BitmapSlice, Error, Result};
 
 impl<F: FileSystem + Sync> Server<F> {
+    #[cfg(feature = "fusedev")]
+    /// Use to send notify msg to kernel fuse
+    pub fn send_inval_entry_notify<S: BitmapSlice>(
+        &self,
+        w: FuseDevWriter<'_, S>,
+        parent: u64,
+        name: &CStr,
```

Comment:
Replace `&CStr` with `std::ffi::CStr`, the line 6-7 can be removed.

Reply:
fixed